docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_3.rst

```diff
@@ -71,6 +71,11 @@ Release notes: Open GEE 5.3.3
          * - 1241
            - geglxinfo should parse all meshes in terrain packets
            - geglxinfo will parse all meshes in terrain packets
+         * - 1552
+           - Make XML object destructors virtual

```

Comment:
Talking about virtual destructors seems a bit technical for release notes. Could we make it more generic like "Properly cleaning up XML objects" or something like that?

Reply:
Fair point. I was going with the title from the issue on GitHub, but I guess that can be changed too. It's fixed now.